statvar_imports/southafrica/budget/SouthAfrica_Budget_pvmap.csv

```diff
@@ -0,0 +1,131 @@
+key,Prop,Value,,,,,,
+Buffalo,observationAbout,wikidataId/Q1002006,observationDate,2018,populationType,EconomicActivity,,
```

Comment:
Is this the combined pvmap? What about the mapping for remaining places Cape Town to wikidataId/Q5465